x-pack/plugins/ml/server/models/data_recognizer/modules/security_windows/ml/v2_windows_anomalous_path_activity_ecs.json

```diff
@@ -0,0 +1,54 @@
+{
+  "job_type": "anomaly_detector",
+  "groups": [
+    "security",
+    "sysmon",
+    "windows",
+    "winlogbeat",
+    "process"
+  ],
+  "description": "This is a new refactored job which works on ECS compatible events across multiple indices. Security: Windows - Looks for activity in unusual paths that may indicate execution of malware or persistence mechanisms. Windows payloads often execute from user profile paths.",
+  "analysis_config": {
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "rare by \"process.working_directory\"",
+        "function": "rare",
+        "by_field_name": "process.working_directory"
+      }
+    ],
+    "influencers": [
```

Comment:
@blaklaybul should we add the `by` field `process.working_directory` to the list of influencers?

Reply:
there is no existing job that this mimics, but I think we should add it for consistency and improved ML ui experience.

Reply:
let me evaluate this one and what it looks like in the UI - it may have been removed due to length

Reply:
So the reason it was removed is that this field is way too long to display in the influencer pane. In my opinion it detracts from UX b/c the user cannot read the values in order to make a selection.